scripts/linux/fv/gitlab-olp-cpp-sdk-functional-test-network.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -ex
+#
+# Copyright (C) 2020 HERE Europe B.V.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+# For core dump backtrace
+ulimit -c unlimited
+
+# Start local server
+node tests/utils/network_server/server.js & export SERVER_PID=$!
+
+# Node can start server in 1 second, but not faster.
+# Add waiter for server to be started. No other way to solve that.
+# Curl returns code 1 - means server still down. Curl returns 0 when server is up
+RC=1
+while [[ ${RC} -ne 0 ]];
+do
+        set +e
+        curl -s http://localhost:8080
+        RC=$?
+        sleep 0.2
+        set -e
+done
+echo ">>> Local Server started for further network test ... >>>"
+
+echo ">>> Start network tests ... >>>"
+$REPO_HOME/build/tests/functional/olp-cpp-sdk-functional-tests  \
+    --gtest_output="xml:$REPO_HOME/reports/olp-functional-test-network-report.xml" \
+    --gtest_filter="NetworkTest.*"
```

Comment:
I don't think it is a good idea to mix function and network tests. This way we need to make sure that the functional test are compilable for all targets which is not always easy. I would prefer that this is a totally separate folder e.g. `functional/network` or similar.

Reply:
moved to network folder and  separate binary.